test/ModeNormal/d i t.test.ts

```diff
@@ -0,0 +1,85 @@
+import * as BlackBox from '../Framework/BlackBox';
+
+suite('Normal: d i t', () => {
+    const testCases: BlackBox.TestCase[] = [
+        {
+            from: '<[]span>content</span>',
+            inputs: 'd i t',
+            to: '<span>[]</span>',
+        },
+        {
+            from: '<span[]>content</span>',
+            inputs: 'd i t',
+            to: '<span>[]</span>',
+        },
+        {
+            from: '<sp[]an>content</span>',
+            inputs: 'd i t',
+            to: '<span>[]</span>',
+        },
+        {
+            from: '<span>content<[]/span>',
+            inputs: 'd i t',
+            to: '<span>[]</span>',
+        },
+        {
+            from: '<span>content</[]span>',
+            inputs: 'd i t',
+            to: '<span>[]</span>',
+        },
+        {
+            from: '<span>content</sp[]an>',
+            inputs: 'd i t',
+            to: '<span>[]</span>',
+        },
+        {
+            from: '<span>content</span[]>',
+            inputs: 'd i t',
+            to: '<span>[]</span>',
+        },
+        {
+            from: '<div>\n<span[]>content</span>\n</div>',
+            inputs: 'd i t',
+            to: '<div>\n<span>[]</span>\n</div>',
+        },
+        {
+            from: '<div>\n<span>content</span[]>\n</div>',
+            inputs: 'd i t',
+            to: '<div>\n<span>[]</span>\n</div>',
+        },
+        {
+            from: '<div[]>\n<span>content</span>\n</div>',
+            inputs: 'd i t',
+            to: '<div>[]</div>',
+        },
+        {
+            from: '<div>\n<span>content</span>\n</[]div>',
+            inputs: 'd i t',
+            to: '<div>[]</div>',
+        },
+        {
+            from: '<div[]><span>content</div></span>',
+            inputs: 'd i t',
+            to: '<div>[]</div></span>',
+        },
+        {
+            from: '<div>[]<span>content</div></span>',
+            inputs: 'd i t',
+            to: '<div>[]</div></span>',
+        },
+        {
+            from: '<div><[]span></div></span>',
+            inputs: 'd i t',
+            to: '<div>[]</div></span>',
+        },
+        {
+            from: '<div><span></div[]></span>',
+            inputs: 'd i t',
+            to: '<div>[]</div></span>',
+        },
+    ];
```

Comment:
@austenc do you think it's worth having tests where the opening/closing tag are on different lines, for completeness?

Reply:
Not a bad idea at all, you're talking about the newline character, yes? `\n` 

I also think some tests where the opening tag itself spans multiple lines would be beneficial. Will take a look and see what I can come up with.

Reply:
Added tests which should cover this. Great suggestion, thanks!